src/puppet-web/puppet-web.ts

```diff
@@ -353,6 +353,12 @@ export class PuppetWeb extends Puppet {
       }))
     })
 
+    // Sending video files is not allowed to exceed 20MB
+    // https://github.com/Chatie/webwx-app-tracker/blob/master/formatted/webwxApp.js#L1115
```

Comment:
Please use the link with a specific commit hash number instead of the latest, because what we want to track is a specific version.

like this: <https://github.com/Chatie/webwx-app-tracker/blob/7c59d35c6ea0cff38426a4c5c912a086c4c512b2/formatted/webwxApp.js#L1115>